src/BoilerPlate/App.cpp

```diff
@@ -117,6 +135,7 @@ namespace Engine
 		double startTime = m_timer->GetElapsedTimeInSeconds();
 
 		// Update code goes here
+		game.UpdateGame(DESIRED_FRAME_TIME, m_height, m_width);
```

Comment:
Remember you need to change this for the delta time for the framerate plot. Also, I'd move away from passing width and height on the Update